File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/C++/s545439898_comments_removed.cpp
#include <iostream>
using namespace std;

int main() {
    int count;
    cin >> count;
    int* givingNumbers = new int[count];

    for (int i = 0; i < count; ++i) {
        cin >> givingNumbers[i];
    }

    for (int i = count - 1; i >= 0; --i) {
        cout << givingNumbers[i];
        if (i != 0) cout << " ";
    }
    cout << endl;

    delete[] givingNumbers;
    return 0;
}
// End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/mad/Java/Go/s545439898_comments_removed.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	countStr := scanner.Text()
	count, err := strconv.Atoi(countStr)
	if err != nil {
		fmt.Println("Error reading count:", err)
		return
	}

	givingNumbers := make([]int, count)
	for i := 0; i < count; i++ {
		scanner.Scan()
		numStr := scanner.Text()
		num, err := strconv.Atoi(numStr)
		if err != nil {
			fmt.Println("Error reading number:", err)
			return
		}
		givingNumbers[i] = num
	}

	for i := count - 1; i >= 0; i-- {
		fmt.Print(givingNumbers[i])
		if i > 0 {
			fmt.Print(" ")
		}
	}
	fmt.Println()
}
// End of Code
